lib/tasks/db.rake

```diff
@@ -1,9 +1,19 @@
-if File.file? 'config/database.yml'
-  require 'active_record_migrations'
+if Module.const_defined?(:ActiveRecord) &&
+   (File.file?('config/database.yml') || ENV['DATABASE_URL']) &&
+   File.file?('config/initializers/active_record.rb')
 
-  ActiveRecordMigrations.configure do |c|
-    c.yaml_config = 'config/database.yml'
+  require 'erb'
+  require './config/initializers/active_record'
+  include ActiveRecord::Tasks
+
+  DatabaseTasks.env = Gris.env
+  DatabaseTasks.db_dir = 'db'
+  DatabaseTasks.migrations_paths = "#{DatabaseTasks.db_dir}/migrate"
+  DatabaseTasks.seed_loader = Gris::SeedLoader.new "./#{DatabaseTasks.db_dir}/seeds.rb"
```

Comment:
DatabaseTasks will use our custom SeedLoader (defined [here](https://github.com/dylanfareed/gris/blob/use-database-tasks/lib/gris/seed_loader.rb) to load the seed.rb file when we run `rake db:seed`